e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/ShellAO.java

```diff
@@ -109,6 +109,14 @@ public ShellAO assertPageAfterCommandNotContainsStrings(String command, String..
         return this;
     }
 
+    public ShellAO assertPageContainsStringsWithRegex(String command, String ... messages) {
+        String log = lastCommandResult(command);
+        Arrays.stream(messages)
+                .forEach(message ->
+                        assertTrue(Pattern.compile(message).matcher(log).find()));
```

Comment:
Could we add the message which message was not found with run id for convenience?